### PR TITLE
Analyst Setup

### PR DESCRIPTION
**Pull request by TOoSmOotH**

### CODE REVIEW

File: setup/so-whiptail [hunk 1]
```diff
@@ -33,6 +33,24 @@ whiptail_airgap() {
...
+
+	read -r -d '' message <<- EOM
+	
+	Welcome to the Security Onion Analyst Workstation install!
+
+	You can choose to join this workstation to an existing grid?
+
+	NOTE: Selecting "no" will enable X Windows and set it to load at boot.
+	EOM
+    echo "whiptail"
```
Comment (dougburks) [inline setup/so-whiptail:48]: Is this a temporary debug line?

File: setup/so-whiptail [hunk 2]
```diff
@@ -33,6 +33,24 @@ whiptail_airgap() {
...
 
+whiptail_analyst_install() {
+
+	[ -n "$TESTING" ] && return
+
+	read -r -d '' message <<- EOM
+	
+	Welcome to the Security Onion Analyst Workstation install!
+
+	You can choose to join this workstation to an existing grid?
```
Comment (dougburks) [inline setup/so-whiptail:44]: Let's change this to:

Would you like to join this workstation to an existing grid?

File: setup/so-whiptail [hunk 3]
```diff
@@ -33,6 +33,24 @@ whiptail_airgap() {
...
+
+	NOTE: Selecting "no" will enable X Windows and set it to load at boot.
```
Comment (dougburks) [inline setup/so-whiptail:46]: I'm guessing that selecting either yes or no still enables the graphical desktop?

File: setup/so-whiptail [hunk 4]
```diff
@@ -33,6 +33,24 @@ whiptail_airgap() {
 	INTERWEBS=$(echo "${INTERWEBS^^}" | tr -d ' ')
 }
 
+whiptail_analyst_install() {
+
+	[ -n "$TESTING" ] && return
```
Comment (dougburks) [inline setup/so-whiptail:38]: This kinda looks like temporary testing code. Do we still need this?

File: setup/so-setup [hunk 5]
```diff
@@ -71,16 +71,49 @@ while [[ $# -gt 0 ]]; do
...
+
+    # Make sure it's CentOS
+    if [[ ! $is_centos ]]; then
+	    echo "Analyst Workstation is only supported on CentOS 7"
+	    exit 1
+	fi
+
+    if ! whiptail_analyst_install; then
+	    # Lets make this a standalone
+		echo "Enabling X Windows and setting it to load at boot"
```
Comment (dougburks) [inline setup/so-setup:104]: Let's change "X Windows" to "graphical desktop".

**Review by dougburks — DISMISSED** [review]

File: setup/so-whiptail (hunk 3 above)
Comment (TOoSmOotH) [reply setup/so-whiptail:46]: Yes but this is only if you chose to do an analyst install from the ISO.

**Review by jertel — APPROVED** [review]

**Pull request merged**